src/cf.data.pre

```diff
@@ -1350,6 +1350,23 @@ endif
 	  #  acl hasWhatMyLoggingDaemonNeeds has request
 	  #  acl hasWhatMyLoggingDaemonNeeds has response
 
+acl aclname at_step step
+	  # match against various request processing steps [fast]
+	  # Valid steps are:
+	  #   GeneratingCONNECT: While generates CONNECT request to a peer
+IF USE_OPENSSL
+	  # The following steps can also be used to match against the
+	  # current step during ssl_bump evaluation.
+	  # Never matches and should not be used outside the ssl_bump context.
```

Comment:
This line is lying, right? Your PR does not add that line, but perhaps we should remove it? If yes, then I would probably remove the whole SslBump text up to the enumeration of specific steps and replace it with something like `The following ssl_bump processing steps are recognized:`.

Reply:
git commit 9a9e0efeb3